### PR TITLE
WAL-264 Support os environment variables

### PR DESCRIPTION
**Pull request by Matrix278**

### CODE REVIEW

**Comment by davidochoa** [comment]
@Matrix278 please make sure you start your new branch from proper base one (master in this case 🙂 )

image

**Comment by Matrix278** [comment]
> @Matrix278 please make sure you start your new branch from proper base one (master in this case  )
> 
> image

Should I start new? Ok, sorry

**Comment by davidochoa** [comment]
Right now it is ok, just merge master to this branch locally

File: README.md [hunk 1]
```diff
@@ -17,12 +17,13 @@ There is test coverage and CI for both linuxish and windows environments, but I
...
 ```shell
-go get github.com/joho/godotenv
+go get github.com/wallester/godotenv
 ```
 
 or if you want to use it as a bin command
+
 ```shell
-go get github.com/joho/godotenv/cmd/godotenv
+go get github.com/wallester/cmd/godotenv
```
Comment (Oxxyg33n) [inline README.md:26]: Link is wrong 🤔

File: README.md [hunk 2]
```diff
@@ -165,7 +166,7 @@ content, err := godotenv.Marshal(env)
 
 Contributions are most welcome! The parser itself is pretty stupidly naive and I wouldn't be surprised if it breaks with edge cases.
 
-*code changes without tests will not be accepted*
+_code changes without tests will not be accepted_
```
Comment (Oxxyg33n) [inline README.md:169]: Here changed too 🤔
